gradle.properties

```diff
@@ -91,10 +91,10 @@ kotlinWasmEnabled=false
 # To know which version, should be used, see compose/frameworks/support/libraryversions.toml
 oel.publication=true
 # Look for `COMPOSE` in libraryversions.toml
-oel.androidx.version=1.4.0
+oel.androidx.version=1.4.3
 # Look for `COMPOSE_MATERIAL3` in libraryversions.toml
 oel.androidx.material3.version=1.0.1
 # No jetpack compose.foundation 1.3.3 published (see https://developer.android.com/jetpack/androidx/releases/compose-foundation)
-oel.androidx.foundation.version=1.4.0
+oel.androidx.foundation.version=1.4.3
 # No jetpack compose.material 1.3.3 published (see https://developer.android.com/jetpack/androidx/releases/compose-material)
```

Comment:
I don't think that comments about `1.3.3` are still relevant. Do not block PR because of that

Reply:
Fixed